apps/frontend/src/style/theme.ts

```diff
@@ -31,237 +44,1488 @@ export const themeFontSize = {
-  fontSizeS: themeFontSize.sText,
-  fontSizeXs: themeFontSize.xsText,
-  lineHeightSText: '14px',
-
-  textColor: themeColor.primaryColor,
-  subtextColor: themeColor.primaryColor,
-  subtextColorActive: themeColor.primaryColor,
-  dataTableTextColor: themeColor.primaryColor,
-  panelToggleBorderColor: themeColor.primaryColor,
-  panelTabWidth: '30px',
-  labelColor: themeColor.primaryColor,
-  labelHoverColor: themeColor.greyDark,
-  labelColorLT: themeColor.greyLight,
-
-  titleTextColor: themeColor.primaryColor,
-  textColorHl: themeColor.greyMedium,
-  textColorHlLT: themeColor.primaryColor,
-  activeColor: themeColor.yellow,
-  activeColorLT: themeColor.yellow,
-  activeColorHover: themeColor.primaryColor,
-  errorColor: themeColor.yellow,
-  logoColor: themeColor.primaryColor,
-
-  errorCardtext: themeColor.yellow,
-  subtextColorCard: themeColor.greyMedium,
-
-  // BUTTON
-  btnFontFamily: fontFamily,
-  primaryBtnBgd: themeColor.primaryColor,
-  primaryBtnActBgd: themeColor.greyDark,
-  primaryBtnColor: themeColor.secondaryColor,
-  primaryBtnActColor: themeColor.greyLight,
-  primaryBtnBgdHover: themeColor.greyDark,
-  primaryBtnRadius: '90px',
-  primaryBtnFontSizeDefault: themeFontSize.sText,
-  primaryBtnFontSizeSmall: themeFontSize.xsText,
-  primaryBtnFontSizeLarge: themeFontSize.mText,
-
-  secondaryBtnBgd: themeColor.primaryColor,
-  secondaryBtnActBgd: themeColor.primaryColor,
-  secondaryBtnColor: themeColor.secondaryColor,
-  secondaryBtnActColor: themeColor.secondaryColor,
-  secondaryBtnBgdHover: themeColor.greyDark,
-  secondaryBtnBorder: '0',
-
-  ctaBtnBgd: themeColor.primaryColor,
-  ctaBtnBgdHover: themeColor.blueLight,
-  ctaBtnActBgd: themeColor.blueLight,
-  ctaBtnColor: themeColor.secondaryColor,
-  ctaBtnActColor: themeColor.secondaryColor,
-
-  linkBtnBgd: themeColor.secondaryColor,
-  linkBtnActBgd: themeColor.greyLight,
-  linkBtnColor: themeColor.primaryColor,
-  linkBtnActColor: themeColor.greyDark,
-  linkBtnActBgdHover: themeColor.greyExtraLight,
-  linkBtnBorder: '1px solid #000000',
-
-  negativeBtnBgd: 'errorColor',
-  negativeBtnActBgd: themeColor.greyLight,
-  negativeBtnBgdHover: themeColor.greyExtraLight,
-  negativeBtnBorder: '0',
-  negativeBtnColor: themeColor.secondaryColor,
-  negativeBtnActColor: themeColor.secondaryColor,
-
-  floatingBtnBgd: themeColor.primaryColor,
-  floatingBtnActBgd: themeColor.greyDark,
-  floatingBtnBgdHover: themeColor.greyDark,
-  floatingBtnBorder: '0',
-  floatingBtnBorderHover: '0',
-  floatingBtnColor: themeColor.secondaryColor,
-  floatingBtnActColor: themeColor.greyExtraLight,
-
-  selectionBtnBgd: 'transparent',
-  selectionBtnActBgd: 'transparent',
-  selectionBtnColor: themeColor.greyLight,
-  selectionBtnActColor: themeColor.yellow,
-  selectionBtnBgdHover: themeColor.yellow,
-  selectionBtnBorder: '1',
-  selectionBtnBorderColor: themeColor.greyLight,
-  selectionBtnBorderActColor: themeColor.yellow,
+export const transition = 'all .4s ease';
```

Comment:
I'm not sure to understand why you had to move all of these properties out of theme object, could you explain it to me?

Reply:
By digging more into the kepler theme to customize it. I realized that I needed more parameters to make some changes related to the request. Thus bringing us closer to the Figma model.

We will be able to be much more precise. Nothing prevents us from subsequently removing options that we do not need.

I find it easier to take full control of Kepler style behaviors.

Reply:
Alright then !